lti_consumer/api.py

```diff
@@ -34,6 +34,10 @@ def _get_or_create_local_lti_config(lti_version, block_location,
     the XBlock to be the source of truth for the LTI version, which is a user-centric perspective we've adopted.
     This allows XBlock users to update the LTI version without needing to update the database.
     """
+    # Replace CCX block locator with block locator to allow CCX launch to work.
+    if hasattr(block_location, 'ccx'):
```

Comment:
What about using the CCXLocator to check if it is a CCX: https://github.com/openedx/ccx-keys? @kuipumu